modules/ossm-rn-new-features.adoc

```diff
@@ -17,13 +17,175 @@ This release adds improvements related to the following components and concepts.
+====
+
+==== OpenShift Service Mesh Console
+
+This release introduces a Developer Preview version of the OpenShift Service Mesh Console, which integrates the Kiali interface directly into the OpenShift Console. For additional informaiton, see link:https://cloud.redhat.com/blog/introducing-the-openshift-service-mesh-console-a-developer-preview[Introducing the OpenShift Service Mesh Console (A Developer Preview)]
+
+==== Cluster-Wide deployment
+
+This release introduces cluster-wide deployment as a technology preview feature. A cluster-wide deployment contains a Service Mesh Control Plane that monitors resources for an entire cluster. The control plane uses a single query across all namespaces to monitor each Istio or Kubernetes resource kind that affects the mesh configuration. Reducing the number of queries the control plane performs improves performance.
+
+===== Configuring cluster-wide deployment
+
+The following example `ConfigMap` object configures a cluster-wide deployment.
+
+There can only be one SMCP in the cluster. Creating the SMCP for cluster-wide deployment requires cluster administrator account privileges. You cannot change a control plane mode from multitenant to cluster-wide (or from cluster-wide to multitenant). If a multitenant control plane already exists, delete it and create a new one.
+
+[source,yaml]
+----
+  apiVersion: maistra.io/v2
+  kind: ServiceMeshControlPlane
+  metadata:
+    name: cluster-wide
+    namespace: istio-system
+  spec:
+    techPreview:
+      controlPlaneMode: ClusterScoped <1>
+    general:
+      logging:
+        componentLevels:
+          default: info
+    version: v2.3
+    runtime:
+      defaults:
+        container:
+          imageRegistry: quay.io/luksa
+          imageTag: latest
+          imagePullSecrets:
+          - name: quay-pull-secret
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+      components:
+        pilot:
+          container:
+            imageName: pilot
+        wasmExtensions.cacher:
+          container:
+            imageName: pilot
+        rateLimiting.rls:
+          container:
+            imageName: ratelimit
+        prometheus:
+          container:
+            imageName: quay.io/maistra/prometheus-rhel8:latest-2.1-qe
+    proxy:
+      runtime:
+        container:
+          imageName: proxyv2
+    ### end changes
+
+    tracing:
+      # change to Jaeger to enable tracing
+      type: None
+    addons:
+      jaeger:
+        name: jaeger
+        install: {}
+      grafana:
+        enabled: false
+        install: {}
+      kiali:
+        name: kiali
+        enabled: true
+        install: {}
+      prometheus:
+        enabled: true
+        install: {}
+  ---
+  apiVersion: maistra.io/v1
+  kind: ServiceMeshMemberRoll
+  metadata:
+    name: default
+  spec:
+    members:
+    - '*' <2>
```

Comment:
The ServiceMeshMemberRoll should probably be in a separate code block, with accompanying text explaining that using the asterisk in `spec.members` ensures that all namespaces are included in the mesh.